test/JDBC/input/storedProcedures/Test-sp_helpdbfixedrole-vu-prepare.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE test_sp_helpdbfixedrole_tbl (DbFixedRole sys.SYSNAME, 
```

Comment:
I think we don't need a table to store the procedure output because this time we don't have a flexible value column like SID in sp_helpsrvrolemember.